dynamic-img/dynamic-img/ui/view/EditionVideo/SelectableLoopVideo.swift

```diff
@@ -28,7 +28,8 @@ struct SelectableLoopVideo: View {
             }
             
             
-        }.frame(width: CGFloat(viewModel.getWidth()), height: CGFloat(viewModel.getHeight()))
+        }
+//        .frame(width: CGFloat(viewModel.getWidth()), height: CGFloat(viewModel.getHeight()))
```

Comment:
Ça vient de là, t'en penses quoi qu'on remet ça comme avant pour l'instant et je l'ajusterai quand je vais faire l'ajustement de la drawZone
```suggestion
        .frame(width: CGFloat(viewModel.getWidth()), height: CGFloat(viewModel.getHeight()))
```